src/lib/modules/Relations/backoffice/components/RelationRemover/RelationRemover.js

```diff
@@ -39,7 +39,12 @@ export default class RelationRemover extends Component {
       >
         <Modal.Header>Confirm removal</Modal.Header>
         <Modal.Content>
-          Are you sure you want to delete this relation?
+          <p>Are you sure you want to delete this relation?</p>
+          <p>
+            Please note that by deleting this {relationType} relation, you
+            delete all the {relationType} relations with the "
+            {related.record_metadata.title}" document.
+          </p>
         </Modal.Content>
         <Modal.Actions>
           <Button onClick={this.handleClose}>No, take me back</Button>
```

Comment:
Do you mind to change all the `No, take me back` with something more formal (there are 2 occurrences in the code)?
Search for all the `handleClose`, I found: `No, take me back`, `Cancel`, `Close`.
I would propose to change **ALL** of them to `Cancel` to be very clear that the operation will be cancelled. WDYT?

Reply:
I agree, it's more consistent.